src/ga4gh/vrs/extras/translator.py

```diff
@@ -10,6 +10,7 @@
 from typing import Optional, Union
 from ga4gh.vrs.dataproxy import create_dataproxy, _DataProxy
 from ga4gh.vrs.extras.decorators import lazy_property
+from hgvs.location import Interval, SimplePosition, AAPosition, BaseOffsetInterval
```

Comment:
I think I would prefer that all the code that is related to these hgvs classes be moved to the hgvs-tools.py module. That was my aim with the prior refactoring work (which was half baked but had good intentions).  Could you consider moving this (or posting a new issue for the future to refactor out the hgvs-specific logic from Translator.pyinto hgvs-tools.py?

Reply:
@larrybabb I'll make a separate issue for this